tests/integration/repo_fork_test.go

```diff
@@ -112,7 +112,8 @@ func TestForkListLimitedAndPrivateRepos(t *testing.T) {
 		req := NewRequest(t, "GET", "/user2/repo1/forks")
 		resp := MakeRequest(t, req, http.StatusOK)
 		htmlDoc := NewHTMLParser(t, resp.Body)
-		assert.Equal(t, 0, htmlDoc.Find(forkItemSelector).Length())
+		// current repository will also use class "flex-item" so we need to subtract 1
+		assert.Equal(t, 0, htmlDoc.Find(forkItemSelector).Length()-1)
```

Comment:
That's really hacky. No better solution?

Reply:
19c0691283